extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -133,7 +130,9 @@ void GetFile::onSchedule(core::ProcessContext *context, core::ProcessSessionFact
     core::Property::StringToInt(value, request_.minSize);
   }
 
-  context->getProperty(PollInterval.getName(), request_.pollInterval);
+  context->getProperty<core::TimePeriodValue>(PollInterval)
+      | utils::map(&core::TimePeriodValue::getMilliseconds)
+      | utils::map([this](std::chrono::milliseconds ms) { request_.pollInterval = ms; });
```

Comment:
this is done elsewhere like this:
```suggestion
  if (const auto poll_interval = context->getProperty<core::TimePeriodValue>(PollInterval)) {
      request_.pollInterval = poll_interval->getMilliseconds();
  }
```
which I find more readable

Reply:
Incorporated your suggestion in b6abaeb